### ISSUE TITLE
documenting macros?

### ISSUE THREAD

**Issue by randy3k** [MichaelHatherly/Lapidary.jl#23]
it seems that there is a problem in documenting macros..

macros.md
```
## Macros

    {docs}
    @rimport
```

julia make.jl
```
ERROR: LoadError: MethodError: `convert` has no method matching convert(::Type{Base.Markdown.MD}, ::Void)
This may have arisen from a call to the constructor Base.Markdown.MD(...),
since type constructors fall back to convert methods.
Closest candidates are:
  Base.Markdown.MD(::Any...)
  call{T}(::Type{T}, ::Any)
  convert{T}(::Type{T}, !Matched::T)
  ...
```

**Comment by MichaelHatherly** [comment]
Is this is Julia 0.4? There's some internal changes between 0.4 and 0.5 in how macros are stored in the docsystem.

Does the following work for you?

```
## Macros

    {docs}
    @rimport(x, args...)

```

i.e. providing the exact signature of the macro.

**Comment by randy3k** [comment]
Yes, I am on julia 0.4, I should have mentioned it. Switching to julia 0.5 does solve the issues.

**Comment by MichaelHatherly** [comment]
05c43b34f91cbc3d648be060743155295e347504 is hopefully enough to handle this issue correctly. @randy3k, if you've got a chance could you confirm whether that commit fixes this for you (on both 0.4 and 0.5).

**Comment by randy3k** [comment]
Thanks, it solves the issues on both 0.4 and 0.5.
